src/app/Fake.DotNet.Paket/Paket.fs

```diff
@@ -181,6 +183,16 @@ let Push setParams =
                     info.Arguments <- sprintf "push %s%s%s%s" url endpoint key (Process.toParam package)) parameters.TimeOut
             if pushResult <> 0 then failwithf "Error during pushing %s." package
 
+/// Pushes all NuGet packages in the working dir to the server by using Paket push.
+/// ## Parameters
+///
+///  - `setParams` - Function used to manipulate the default parameters.
+let Push setParams =
+    let parameters : PaketPushParams = PaketPushDefaults() |> setParams
```

Comment:
This is unused?

Reply:
I think we can just remove this line.

Reply:
This is used to get the `WorkingDir` for the globbing pattern below.

I can change the functions so we don't evaluate `setParams` twice.

Reply:
Oh wow apparently I'm blind, thanks ;)

So you suggest to `(fun _ -> parameters)`, correct? Yes we can do that.